frontend/views/components/Contribution.vue

```diff
@@ -51,7 +51,7 @@ transition(name='replacelist')
       @click='handleEditClick'
       data-test='buttonEditNonMonetaryContribution'
     )
-      i.icon-pencil-alt(aria-hidden='true')
+      i.icon-pencil-alt.is-prefix
```

Comment:
You meant to remove `(aria-hidden='true')` here?

Reply:
Yes, by default the icons aren't read by SR, I tested it myself